README.rst

```diff
@@ -229,3 +229,30 @@ Appendix B: Adding No-Op Stuff to Test Sandbox Deploys
 ----------------------------------------------------------
 
 Let's try this.
+
+
+Subscriptions
+-------------
+
+This payment MFE has introduced an additional React route, ``payment.stage.edx.org/subscription``, specifically designed to handle recurring payments. Currently, we are only permitting one payment method to manage recurring purchases, namely stripe subscriptions.
+
+Development
+-----------
+
+All functionality related to subscriptions is built under the ``src/subscription`` directory. It seeks to optimize the utilization of existing functionality to implement this new feature, adding any necessary enhancements on top of it. The following list outlines key points where it diverges from the ``src/payment`` flow:
+
+- Main ``src/index.scss`` includes ``src/subscription/index.scss`` after ``src/payment/index.scss`` allowing it to override the payment page styles.
+- ``src/subscription/i18n-protected-messages`` overrides the *Transifex* messages with the locally (hardcoded) defined messages, specifically for ``es-419`` locale.
+- ``src/subscription/test-utils`` defines the new test wrapper with ``react-testing-library``. We use ``react-testing-library`` to write unit tests for ``src/subscription`` code.
+- ``src/subscription/alerts`` defines its new set of alerts specific to ``Subscription`` page.
+- ``SUBSCRIPTIONS_BASE_URL`` exposes different endpoints for ``GET`` and ``POST`` subscription CTAs.
```

Comment:
Suggestion: `exposes different subscriptions backend endpoints`, it's implied but just a reminder these refer to our subscriptions BE repo

Reply:
done. https://2u-internal.atlassian.net/wiki/spaces/RS/pages/551321788/How+to+handle+b2c+recurring+payments+on+the+payment+checkout+page